### PR TITLE
Feature/hero unit

### PR DESCRIPTION
**Pull request by eliashedlund**
Added the hero component to the styleguide and made it work better on small devices.

screen shot 2015-06-25 at 14 43 14

screen shot 2015-06-25 at 14 42 33

### CODE REVIEW

File: src/mnd-bootstrap/components/_hero.scss [hunk 1]
```diff
@@ -0,0 +1,96 @@
+/*doc
```
Comment (mnd-hound) [inline src/mnd-bootstrap/components/_hero.scss]: Use `//` comments everywhere

File: src/mnd-bootstrap/components/_hero.scss [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+```
+*/
+
+$space-x5: $space * 5;
+
+.hero-content {
+  border: 1px solid $gray-lighten;
+  border-radius: 4px;
+  margin: 15px auto 0;
+  padding: 15px;
```
Comment (mnd-hound) [inline src/mnd-bootstrap/components/_hero.scss]: Properties should be ordered border, border-radius, margin, max-width, padding

**Comment by jorkas** [comment]
@eliashedlund Looks good, but isn't the hero replaced by jumbotron in bootstrap 3? If thats the case maybe we can use the jumbotron instead?

**Comment by eliashedlund** [comment]
@jorkas Oh I didn't know there was an actual hero component in bootstrap earlier. Thought it was just a new component we had created. At least that's what's been said when I've asked about it and that we should keep it separate from the jumbotron. But if we don't need the jumbotron as it looks now, I agree that we probably should use that instead.

File: src/mnd-bootstrap/components/_jumbotron.scss [hunk 3]
```diff
@@ -4,83 +4,95 @@
...
+    text-align: center;
+
+    @media screen and (min-width: $screen-sm-min) {
+      margin-bottom: $space-larger;
     }
 
-    button {
-      float: left;
+    p {
+      font-size: $font-size-base;
```
Comment (mnd-hound) [inline src/mnd-bootstrap/components/_jumbotron.scss]: Properties should be ordered color, font-size, font-weight

**Comment by eliashedlund** [comment]
I've redesigned the jumbotron to look like the hero component and removed the hero component. There will be additional design updates for this in the near future.

**Pull request merged**